boards/nordic/nrf54h20dk/Kconfig.defconfig

```diff
@@ -6,13 +6,19 @@ if BOARD_NRF54H20DK_NRF54H20_CPUAPP
 config BT_HCI_IPC
 	default y if BT
 
+config MAX_THREAD_BYTES
+	default 3 if USERSPACE
+
 endif # BOARD_NRF54H20DK_NRF54H20_CPUAPP
 
 if BOARD_NRF54H20DK_NRF54H20_CPURAD
 
 config BT_CTLR
 	default y if BT
 
+config MAX_THREAD_BYTES
+	default 3 if USERSPACE
```

Comment:
isn't it a duplicate of what you already added?

Reply:
No it is not, one is for the application domain and one is for the radio domain. They are guarded by BOARD_NRF54H20DK_NRF54H20_CPURAD  and BOARD_NRF54H20DK_NRF54H20_CPUAPP respectively.

Reply:
i see it now, thanks